### PR TITLE
ACS-2827: Enable TAS CMIS (via CMIS API) Elastic Tests

### PR DESCRIPTION
**Pull request by KikoUK**

### CODE REVIEW

**Review by tpage-alfresco — APPROVED** [review]
As much as I hate the proliferation of these docker-compose files - this looks good to me :yum:

**Review by gevagel — APPROVED** [review]
LGTM 👍

File: tests/environment/.env [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
 SYNC_SERVICE_TAG=3.7.0
 ES_TAG=7.10.1
+TRANSFORM_ROUTER_TAG=1.5.2
+TRANSFORM_CORE_AIO_TAG=2.5.7
+SHARED_FILE_STORE_TAG=0.16.1
+ACS_NGINX_TAG=3.0.0
+ELASTICSEARCH_TAG=7.10.1
+KIBANA_TAG=7.10.1
+LIVE_INDEXING_TAG=3.1.1-A1
+LIVE_REINDEXING_TAG=3.1.1-A1
```
Comment (dsibilio) [inline tests/environment/.env:15]: Is this `LIVE_REINDEXING_TAG` variable actually used anywhere?

File: tests/environment/.env [hunk 2]
```diff
@@ -1,9 +1,21 @@
+ALFRESCO_TAG=latest
 SOLR6_TAG=2.0.3
 POSTGRES_TAG=13.3
 ACTIVEMQ_TAG=5.16.1
 AIMS_TAG=1.2
 SYNC_SERVICE_TAG=3.7.0
 ES_TAG=7.10.1
+TRANSFORM_ROUTER_TAG=1.5.2
```
Comment (dsibilio) [inline tests/environment/.env:8]: Isn't the `TRANSFORM_ROUTER_TAG` dynamically evaluated [here](https://github.com/Alfresco/alfresco-community-repo/blob/master/packaging/tests/scripts/start-compose.sh#L31)?

**Review by dsibilio — APPROVED** [review]
LGTM! 👍 Maybe could get rid of some extra variables as pointed out in review comments, or maybe I'm missing something. Good work either way 👏

**Review by damianujma — APPROVED** [review]

File: tests/environment/.env (hunk 1 above)
Comment (KikoUK) [reply tests/environment/.env:15]: Good spot. I removed the reindexing d-c as it doesn't seem to be needed and forgot the tag 👍

File: tests/environment/.env (hunk 2 above)
Comment (KikoUK) [reply tests/environment/.env:8]: I added it in the .env for local runs without having to use the start-compose script

**Pull request merged**